upgrade-guides/v2.0.0.md

```diff
@@ -102,7 +102,9 @@ const appHistory = useScroll(useRouterHistory(createBrowserHistory))();
 
 ## Changes to `this.context`
 
-Only an object named `router` is added to context. Accessing `this.context.history`, `this.context.location`, and `this.context.route` are all deprecated. This new object contains the methods available from `history` (such as `push`, `replace`) along with `setRouteLeaveHook`.
+**NOTE: v2.4.0 and higher include a higher-order component `withRouter` that is now the preferred way of accessing the `router` object.** Read [the v2.4.0 upgrade guide](/upgrade-guides/v2.4.0.md) for more details.
```

Comment:
I'd say "recommended" instead of "preferred". Not sure if it makes a difference.